llvm/include/llvm/Passes/MachinePassRegistry.def

```diff
@@ -146,6 +147,7 @@ MACHINE_FUNCTION_PASS("opt-phis", OptimizePHIsPass())
 MACHINE_FUNCTION_PASS("peephole-opt", PeepholeOptimizerPass())
 MACHINE_FUNCTION_PASS("phi-node-elimination", PHIEliminationPass())
 MACHINE_FUNCTION_PASS("print", PrintMIRPass())
+MACHINE_FUNCTION_PASS("print<livedebugvars>", LiveDebugVariablesPrinterPass(dbgs()))
```

Comment:
Not here, but we should fix these to use consistent - word separation

Reply:
This should not be using dbgs - the existing uses should also be errs

Reply:
> This should not be using dbgs - the existing uses should also be errs

Yes. @optimisan can you post a PR to change the `dbgs` to `errs` in all similar instances?

Reply:
Can just use errs here in the first place, don't need to wait for the other PR